hoomd/write/gsd.py

```diff
@@ -46,15 +38,33 @@ class GSD(Writer):
     over time. `GSD` can also store operation-specific state information
     necessary for restarting simulations and user-defined log quantities.
 
+    Valid file open modes:
+
+    +------------------+---------------------------------------------+
+    | mode             | description                                 |
+    +==================+=============================================+
+    | ``'wb'``         | Open the file for writing. Creates the file |
+    |                  | if needed, or overwrites an existing file.  |
+    +------------------+---------------------------------------------+
+    | ``'xb'``         | Create a GSD file exclusively and opens it  |
+    |                  | for writing.                                |
+    |                  | Raise an exception when it already exists.  |
+    +------------------+---------------------------------------------+
+    | ``'ab'``         | Appends to the file when it already exists. |
+    |                  | Creates the file when it does not exist.    |
```

Comment:
Tense is inconsistent in this table: create/creates, raise/raises.

Reply:
fixed